### DISCUSSION TITLE
Scan qr code to login

### DISCUSSION THREAD

**Discussion by dfang** [community/community#13008] [Mobile]
scan qr code to login github.com on browser like whatsapp web

**Comment by aconsuegra** [comment]
Hi @dfang, thanks for your feature request! I will send it over to our product team and we will evaluate it soon. Keep the feedback coming!

**Reply by ianzone** [reply]
How is it going after 3 years?

**Comment by server-mm404** [comment]
> scan qr code to login github.com on browser like whatsapp web
